javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const filteredValues = [];
+  for (let i = 0; i < values.length; i++) {
+    console.log(values[i]);
```

Comment:
`console.log` is only used for testing / debugging purposes, so we want to always remove these before submitting our exercises! Unless we specifically ask for you to include them in the instructions

Reply:
Oh yeah, I was logging the loops to figure out what was happening and forgot they were in there. I'll remove and commit again!